pipeline_documentation.md

```diff
@@ -106,14 +105,14 @@ on installation and usage please see [here](README.md).
 
 ## Description of workflow steps
 
-> The workflow consists of four Snakemake files: A main `Snakefile` and an
-individual Snakemake file for each step in the workflow (the file preparation,
-the reads mapping, the miRNA quantification and the ASCII-style alignment
-pileups generation). The main `Snakefile` contains the configuration file
-validation along with the inclusion of the sub-workflows. Individual steps of
-the workflow are described briefly, and links to the respective software
-manuals are given. Parameters that can be modified by the user (via the samples
-table and the configuration file) are also described.
+> The workflow consists of five Snakemake files: A main `Snakefile` and an
+> individual Snakemake file for each step in the workflow (the genome resources
```

Comment:
I wouldn't call this "step" - outside of Snakemake (where it's called "rule"), a workflow step is one particular job/task, so it may be confusing. Maybe use "part". Or just don't say anything at all and just say "individual Snakemake file each for the genome resources preperation, the reads mapping [...]".